config/base/operator-controller/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: olmv1-system
+namePrefix: operator-controller-
```

Comment:
Could we keep the comments to clarify how it is used?

Reply:
I actually don't think the comments are useful.